HabitTracker.w0lvesvvv/HabitTracker.w0lvesvvv/HabitRecord.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HabitTracker.w0lvesvvv
+{
+    public class HabitRecord
+    {
+        public string record_habit_name_nv { get; set; }
```

Comment:
as per our [code conventions](https://thecsharpacademy.com/article/58), property names should be pascal case